orcid-core/src/main/java/org/orcid/core/manager/impl/ProfileWorkManagerImpl.java

```diff
@@ -189,6 +193,18 @@ public Work createWork(String orcid, Work work) {
         return jpaJaxbWorkAdapter.toWork(profileWorkEntity);
     }
 
+    private void validateWork(Work work) {
+        WorkTitle title = work.getWorkTitle();
+        if (title == null || title.getTitle() == null || StringUtils.isEmpty(title.getTitle().getContent())) {
+            throw new OrcidValidationException("Invalid Title: title cannot be null nor emtpy");
+        }
+
+        if (work.getWorkExternalIdentifiers() == null || work.getWorkExternalIdentifiers().getWorkExternalIdentifier() == null
+                || work.getWorkExternalIdentifiers().getWorkExternalIdentifier().isEmpty()) {
+            throw new OrcidValidationException("Invalid work: Works added using message version 1.2_rc5 or greater must contain at least one external identifier");
```

Comment:
@nitw-shobhit dont hardcode the exception, add a i18n key and the locale manager to get the error message